libraries/ms-common/inc/stm32f0xx/gpio_cfg.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
We should standardize on _mcu or _cfg. Alternatively, if there's a valid reason to have both, we should define what each one represents.